presto-iceberg/src/test/java/com/facebook/presto/iceberg/hive/TestIcebergDistributedHiveLockDisabled.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.iceberg.hive;
+
+import com.facebook.presto.hive.HdfsContext;
+import com.facebook.presto.hive.HiveColumnConverterProvider;
+import com.facebook.presto.hive.metastore.MetastoreContext;
+import com.facebook.presto.iceberg.HiveTableOperations;
+import com.facebook.presto.iceberg.IcebergHiveTableOperationsConfig;
+import com.facebook.presto.iceberg.ManifestFileCache;
+import com.facebook.presto.metadata.CatalogManager;
+import com.facebook.presto.spi.ColumnMetadata;
+import com.facebook.presto.spi.ConnectorId;
+import com.facebook.presto.spi.ConnectorSession;
+import com.google.common.cache.CacheBuilder;
+import com.google.common.collect.ImmutableMap;
+import org.apache.iceberg.BaseTable;
+import org.apache.iceberg.PartitionSpec;
+import org.apache.iceberg.Table;
+import org.apache.iceberg.TableMetadata;
+import org.apache.iceberg.Transaction;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static com.facebook.presto.common.type.IntegerType.INTEGER;
+import static com.facebook.presto.hive.metastore.MetastoreUtil.getMetastoreHeaders;
+import static com.facebook.presto.hive.metastore.MetastoreUtil.isUserDefinedTypeEncodingEnabled;
+import static com.facebook.presto.iceberg.IcebergAbstractMetadata.toIcebergSchema;
+import static com.google.common.io.Files.createTempDir;
+import static org.apache.iceberg.TableMetadata.newTableMetadata;
+import static org.apache.iceberg.Transactions.createTableTransaction;
+import static org.testng.Assert.assertEquals;
+
+@Test
+public class TestIcebergDistributedHiveLockDisabled
+        extends TestIcebergDistributedHive
+{
+    public TestIcebergDistributedHiveLockDisabled()
+    {
+        // This property will turn off locks whenever committing to a HMS. This can break existing functionality so we
+        // are rerunning all the hive tests with locks disabled to verify nothing breaks.
```

Comment:
Do we need to add these tests forever, or is it just a quality assurance measure for the time being?